src/test/java/com/capitalone/dashboard/collector/PullRequestCollectorTest.java

```diff
@@ -89,30 +90,31 @@ public void testGetPullRequests() throws IOException, URISyntaxException, Encryp
                         "https://company.com/rest/api/1.0/projects/myproject/repos/myrepository/pull-requests");
         given(bitbucketApiUrlBuilder.buildPullRequestApiUrl(repoUrl)).willReturn(value);
 
-        given(scmHttpRestClient.makeRestCall(uri1, repo.getUserId(), "password"))
+        given(scmHttpRestClient.makeRestCall(uri1, "abcdefgh","abcdefghijklmnopqrstuvwxyz1234567"))
                 .willReturn(new ResponseEntity<>(prResponseTestData, HttpStatus.OK));
-        given(scmHttpRestClient.makeRestCall(uri2, null, ""))
+        given(scmHttpRestClient.makeRestCall(uri2, "abcdefgh",""))
                 .willReturn(new ResponseEntity<>(prResponseTestData, HttpStatus.OK));
 
 
         // when
-        int pullCount = pullRequestCollector.getPullRequests(repo, "OPEN");
+        int pullCount = pullRequestCollector.getPullRequests(repo, "OPEN", "abcdefgh","abcdefghijklmnopqrstuvwxyz1234567");
 
         // then
         assertEquals(pullCount, 3);
 
     }
 
     @Test
-    public void testGetPullRequests_RecentlyUpdated() throws IOException, URISyntaxException, EncryptionException {
-        given(settings.getKey()).willReturn("abcdefghijklmnopqrstuvwxyz1234567");
-        String encPassword = Encryption.encryptString("password", settings.getKey());
-        given(settings.getPassword()).willReturn(encPassword);
+    public void testGetPullRequests_RecentlyUpdated() throws IOException, URISyntaxException, Exception {
+//        given(settings.getKey()).willReturn("abcdefghijklmnopqrstuvwxyz1234567");
```

Comment:
please remove all commented lines of code

Reply:
done